sormas-ui/src/main/java/de/symeda/sormas/ui/contact/MergeGuideLayout.java

```diff
@@ -0,0 +1,83 @@
+package de.symeda.sormas.ui.contact;
+
+import com.vaadin.shared.ui.ContentMode;
+import com.vaadin.ui.Label;
+import com.vaadin.ui.VerticalLayout;
+
+import de.symeda.sormas.api.i18n.Captions;
+import de.symeda.sormas.api.i18n.I18nProperties;
+import de.symeda.sormas.api.i18n.Strings;
+import de.symeda.sormas.ui.utils.CssStyles;
+
+public class MergeGuideLayout extends VerticalLayout {
```

Comment:
this is an exact copy of the case layout 
please either introduce an abstraction or an object that contains these strings and provide this object for the 1 MergeGuideLayout